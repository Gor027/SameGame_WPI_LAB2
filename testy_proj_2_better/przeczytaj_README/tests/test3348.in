A AAAAxAA xAA AxAxx x xxAA xx AA xxxAxA xAxxAAA Ax A
A AxAAxAx  xAxA AAAxx A x AA  AAx
