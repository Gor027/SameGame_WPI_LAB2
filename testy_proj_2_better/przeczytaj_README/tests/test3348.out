 xAA AA A Axx xx A  x AA
xxAxxxAxAxAAAxxAxxAAAxAAxxxxAxAxAxxAAAAxA
